oak/server/rust/oak_runtime/src/node/grpc/server.rs

```diff
@@ -378,6 +382,16 @@ impl GrpcRequestHandler {
                         m.data
                     })
                 })
+                .and_then(|response| {
+                    // Decode [`prost_types::Any`] message.
```

Comment:
Which one is the `Any` message exactly? I don't think there should be one.

Reply:
Thanks!
Deleted